doc/mgmt/SONiC_OC_MGMT_Interface_HLD.md

```diff
@@ -0,0 +1,261 @@
+# Feature Name
+Openconfig support for Physical and Management interfaces via openconfig-interfaces.yang.
+# High Level Design Document
+#### Rev 0.1
+
+# Table of Contents
+  * [List of Tables](#list-of-tables)
+  * [Revision](#revision)
+  * [Scope](#scope)
+  * [Definition/Abbreviation](#definitionabbreviation)
+
+# List of Tables
+[Table 1: Abbreviations](#table-1-abbreviations)
+
+# Revision
+| Rev |     Date    |       Author       | Change Description                |
+|:---:|:-----------:|:------------------:|-----------------------------------|
+| 0.1 | 09/09/2019  |   Ravi Vasanthm     | Initial version                   |
+
+# About this Manual
+This document provides general information about openconfig support for Physical and Management interfaces handling in SONIC.
+
+# Scope
+This document describes the high level design of openconfig support for Physical and Management interfaces handling feature. Call out any related design that is not covered by this document
+
+# Definition/Abbreviation
+
+### Table 1: Abbreviations
+| **Term**                 | **Meaning**                         |
+|--------------------------|-------------------------------------|
+| MGMT Intf                     | Management Interface                     |
+
+# 1 Feature Overview
+This feature will provide config set/get and status get support for Physical and Management interfaces according to the openconfig-interfaces.yang data model via CLI, REST, and gNMI.
+
+https://github.com/project-arlo/sonic-mgmt-framework/blob/master/models/yang/openconfig-interfaces.yang
+
+## 1.1 Requirements
+
+
+### 1.1.1 Functional Requirements
+
+1. Provide CLI, REST and GNMI support for configuring and displaying physical and management interfaces attributes.
+2. Enhance existing implementation of interfaces yang to include physical and management interfaces handling.
```

Comment:
Could you please clarify what is "interfaces yang" ? This is unclear...
Do you mean YANG models ?
If yes, please update the text - and use the proper capitalization for YANG
(Yet Another Next Gen :-) )

Reply:
Addressed